system_reports/MACOS/INTEL/osx-xcode-13.1.x.log

```diff
@@ -2130,7 +2185,7 @@ faraday-rack (1.0.0)
 faraday-retry (1.0.3)
 faraday_middleware (1.2.0)
 fastimage (2.2.6)
-fastlane (2.207.0, 2.206.2)
+fastlane (2.208.0, 2.207.0, 2.206.2)
```

Comment:
There are more fastlane versions installed. Just curios, did we deliberately installed new version or this is just an additional dependency from somewhere?

Reply:
@fehersanyi-bitrise , @pgyula - can you please answer this?

Reply:
added dependencies from some magical place